files/build_templates/per_namespace/bgp.service.j2

```diff
@@ -1,16 +1,29 @@
 [Unit]
 Description=BGP container
+{% if multi_instance == 'true' %}
+Requires=database@%i.service
 Requires=updategraph.service
+After=database@%i.service
 After=updategraph.service
+{% else %}
+Requires=updategraph.service
+After=updategraph.service
+{% endif %}
```

Comment:
We could possibly simplify all these docker service file templates and eliminate duplicate lines. For example:

```suggestion
{% if multi_instance == 'true' %}
Requires=database@%i.service
After=database@%i.service
{% endif %}
Requires=updategraph.service
After=updategraph.service
```

Reply:
Modified as per suggestion